### ISSUE TITLE
Star Wars Outlaws graphical bug

### ISSUE THREAD

**Issue by hpeter1994** [HansKristian-Work/vkd3d-proton#2116]
### Software information
Star Wars Outlaws, Snow effect glitchy. Tried multiple graphical settings, with framegen, without framegen, max settings minimum settings, even switched to the integrated AMD GPU but the issue still persists. I have not found any mention of this glitch on the internet so i'm fairly certain it does not come up on windows. 
I'm using Bottles, and tried it with the default soda-9.0-1, lutris-7.2-2, GE-Proton9-12 and GE-Proton9-13 runners using dxvk-2.4 and vkd3d-proton-2.13

Video of the issue:
https://youtu.be/aSW03WYawUM

### System information
- GPU:  AMD Radeon RX 7900 GRE and AMD Radeon Graphics (AMD Ryzen 5 7600)
- Driver: latest amdgpu from arch repos. (Mesa 24.2.1-arch1.1)
- Wine version: GE-Proton9-13
- VKD3D-Proton version: vkd3d-proton-2.13

### Log files
[logfile.txt](https://github.com/user-attachments/files/16887578/logfile.txt)

**Comment by Blisto91** [comment]
You seem to be running the amdvlk driver instead of mesa radv as you have written in the issue. I assume that is unintentional

**Comment by hpeter1994** [comment]
Well, its certanly not intentional on my part, i copied the driver version from glxinfo, so maybe that was the mistake. I have been messing around a lot when i tried to get gpu-passthrough working, and at the end i gave up when i realized i can run most of my games througt proton.

Game has been running perfectly fine for this last week, ocasional crashes, but no major issues, except this graphical bug on the newly unlocked planet.

**Comment by Blisto91** [comment]
Yes it isn't unusual to have both installed

**Comment by hpeter1994** [comment]
I have been using the default bottles runner (soda i think), until yesterday when i tried installing other stuff trying to fix the issue. GE 12 also ran the game, and that one certanly had the older vkd3d-proton version.
So if i'm apperantly running it with amdvlk, then the mentioned crashes must be related to radv somehow.

[20240904_13h05m36s.save.zip](https://github.com/user-attachments/files/16888335/20240904_13h05m36s.save.zip)
Save should be right next to a glitchy snowpatch, but the whole area is full of it.

**Comment by mbriar** [comment]
Install `vulkan-radeon lib32-vulkan-radeon`, uninstall `amdvlk lib32-amdvlk`. Unintentionally using amdvlk is issue number 1 for Arch linux users with amd gpus.

**Comment by dominikzogg** [comment]
> Yes it isn't unusual to have both installed
> 
> Edit: first noticed now that you were using GE 13 which had the latest vkd3d-proton fixes so nvm to the vkd3d-proton master part.
> 
> Edit 2: can you share your save game?

No, GE 13  doesn't contain the fix (at least it has not when used with bottles) but compiling vkd3d using master does.

**Comment by Blisto91** [comment]
GE 13 contains all the work that has been dedicated specifically to Outlaws yes

**Comment by dominikzogg** [comment]
@Blisto91 so it seems an issue with bottles, or me understanding their settings: https://github.com/bottlesdevs/Bottles/issues/3506

**Comment by HansKristian-Work** [comment]
This should be fixed on master now I think ...
